### PR TITLE
UI changes   add task page

### PR DESCRIPTION
**Pull request by JasonRenBang**
bring all colors back and set work task input area as default

### CODE REVIEW

**Review by Ben0189 — DISMISSED** [review]

**Review by PeterOu8 — CHANGES_REQUESTED** [review]
Good work! However, you have the label text colours and the textarea place holder colours are mismatched with the figma prototype. Could you please change those as well?

File: blotztask-ui/src/app/add-task/page.tsx
Comment (PeterOu8) [inline blotztask-ui/src/app/add-task/page.tsx:1]: e.g. add the corresponding colour in the tailwind config file and use them in your page. #6021D6 for work label text

Comment (JasonRenBang) [reply blotztask-ui/src/app/add-task/page.tsx:1]: how about this time

**Review by PeterOu8 — DISMISSED** [review]
LGTM!

**Review by PeterOu8 — APPROVED** [review]
All g to go

**Pull request merged**